### ISSUE TITLE
Stubbing services

### ISSUE THREAD

**Issue by tchan** [ember-learn/guides-source#78]
https://guides.emberjs.com/release/testing/testing-components/#toc_stubbing-services

As per discussion on slack (hopefully it hasn't disappeared) https://embercommunity.slack.com/archives/C045C9ABM/p1528120489000077 , the code snippets on updating values on a service is outdated and perhaps should be something like (if there's a more ember-matic way i'd be curious to know as well)

```
    const locationService = this.owner.lookup('location-service');
    locationService.set('city', 'Melbourne');
```

The code snippets also look a bit funky as well
![ss](https://puu.sh/Azk0I.png)

@jenweber

**Comment by sivakumar-kailasam** [comment]
This is now fixed. thanks for reporting it :)

**Comment by acorncom** [comment]
@tchan would you have time to update the guides here? Would love to get this cleaned up ...

**Comment by spruce** [comment]
This is resolved as of #218 isn't it?

**Comment by sivakumar-kailasam** [comment]
Yes it was. Thanks for the update.
